lib/mixpanel-ruby/consumer.rb

```diff
@@ -3,7 +3,13 @@
 require 'json'
 
 module Mixpanel
-  class ConnectionError < IOError
+  class MixpanelError < Error
```

Comment:
Should this be `MixpanelError < Exception`? It seems to have broke the travis build.